src/requirements.ts

```diff
@@ -26,8 +27,8 @@ export interface RequirementsData {
 interface ErrorData {
   message: string;
   label: string;
-  openUrl: Uri;
-  replaceClose: boolean;
+  command: string;
```

Comment:
TypeScript tells me this `ErrorData` interface is not used anymore, we might as well remove it.